fluentd/tests/util.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from .common import FLUENTD_VERSION
+
+requires_1_9 = pytest.mark.skipif(
+    FLUENTD_VERSION is None or FLUENTD_VERSION != '1.9.3',
```

Comment:
Should be `>=` instead of  `!=` ? That will ease maintenance if we add new versions to tox.ini.

Reply:
I'm wary of using >= with a string
```
>>> '1.9.3' > '1.10.3'
True
```

Reply:
You can maybe use this:

```python
>>> from packaging import version
>>> version.parse("1.9.3") >= version.parse("1.10.3")
False
```

Reply:
We could even build a pair of `@requires_below_1_8_0`/`@requires_above_1_8_0` since this is the exact pivot version for the things we're testing (we don't actually care that we use 1.9.0 vs 0.12, it's only important that there's a version before 1.8.0 and one after).